torch/_inductor/runtime/triton_heuristics.py

```diff
@@ -490,7 +491,8 @@ def _precompile_config(self, cfg: Config, warm_cache_only: bool):
             compile_kwargs = compile_meta
 
         if warm_cache_only:
-            binary = triton.compile(*compile_args, **compile_kwargs)
+            with _WaitCounter("pytorch.triton_heuristics.compile"):
```

Comment:
FYI this will almost always be run in a separate process so you might not be getting what you want.